Documentation/getting_started/3_adding_content.md

```diff
@@ -15,11 +15,12 @@ You should now see a MonoGame Pipeline Tool window open up. In case it didn't ge
 ![MonoGame Pipeline Tool](images/getting_started/3_pipeline_tool.png)
 
 Your game content is managed from this external tool. You can add content to your game in one of the following ways:
- - **Add Existing Item** toolbar button
- - **Edit > Add > Existing Item...** menu button
- - **right-click > Add > Existing Item...** context menu
- 
-In our case lets user the **Add Existing Item** toolbar button.
+
+- **Add Existing Item** toolbar button
+- **Edit > Add > Existing Item...** menu button
+- **right-click > Add > Existing Item...** context menu
+
+In our case let's user the **Add Existing Item** toolbar button.
```

Comment:
let's **use**

Reply:
@Jjagg I'll just add you as admin on my fork so you can modify instead of doing this the long way around.